client/src/main/java/building/FleetDropController.java

```diff
@@ -0,0 +1,175 @@
+package building;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.event.EventHandler;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.effect.DropShadow;
+import javafx.scene.input.*;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+import model.*;
+
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ResourceBundle;
+
+public class FleetDropController implements Initializable {
+
+    private static final int FIELD_SIZE = 30;
+    private static final int GRID_SIZE = 10;
+
+    @FXML
+    GridPane shipBoard;
+
+    @FXML
+    public Pane port;
+
+    @FXML
+    public Rectangle ship4, ship3, ship3a, ship2, ship2a, ship2b, ship1, ship1a, ship1b, ship1c;
```

Comment:
for me it is unreadable, when more then 1 variable is initialize int the 1 line http://www.informit.com/articles/article.aspx?p=2133373&seqNum=2